src/renderer/api/kube-watch-api.ts

```diff
@@ -32,8 +32,6 @@ export interface IKubeWatchLog {
 @autobind()
 export class KubeWatchApi {
   @observable context: ClusterContext = null;
-  @observable subscribers = observable.map<KubeApi, number>();
-  @observable isConnected = false;
 
   contextReady = when(() => Boolean(this.context));
 
```

Comment:
Maybe this isn't for this PR, but what exactly is the constructor "supposed" to be doing here? But besides that, what is with the habit of calling async functions in a constructor?

Reply:
Removed, should have been removed already in #2095 .